### DISCUSSION TITLE
Can I save dependent files to the specified directory?

### DISCUSSION THREAD

**Discussion by kingmo888** [pyinstaller/pyinstaller#7271] [Help]
In advance, there is no problem with the current default mode.
By default, the files generated by packaging are similar to the following formats:

dist
  -exefolder
    --folder1
    --folder2
    --folder3
    --folder4
    --...
    --folderN
    --ExeFile.exe
    --xxx.dll
    --xxx.dll
    --xxx.dll
    --xxx.dll

Whether these dlls and folders can be uniformly placed in the specified subfolders (such as dist/exefolder/python) under the top-level directory, which will make the packaged structure cleaner.

**Comment by rokm** [comment]
No, the file and directory layout is fixed and cannot be changed. There are different search path mechanisms involved that depend on the layout being as it is.

**Reply by kingmo888** [reply]
What I said can't be reached. I understand. thank you.
